packages/x-components/src/x-modules/search/components/banner.vue

```diff
@@ -1,15 +1,30 @@
 <template>
   <a
+    v-if="banner.url && !imageError"
```

Comment:
Now that this is handling the case where the image load errors, there should be a test for it.

Reply:
done